aiohttp/web.py

```diff
@@ -290,7 +291,7 @@ async def _run_app(
     *,
     host: Optional[Union[str, HostSequence]] = None,
     port: Optional[int] = None,
-    path: Optional[str] = None,
+    path: Optional[Union[str, Path]] = None,
```

Comment:
@Dreamsorcerer I wonder if this should've been `os.PathLike` instead...

Reply:
Hmm, seems we even have a PathLike in typedefs.py.

Reply:
I'm already on it. Updating a couple of other places too.
#6876

Reply:
You update this pull request ? https://github.com/aio-libs/aiohttp/pull/6874

Reply:
> You update this pull request ? #6874

No, backports must be as close to the original patch as possible, for traceability.